requirements.txt

```diff
@@ -23,6 +23,8 @@ timm==0.6.13
 autocare-dlt==0.2.4
 
 # etc
+numpy<=1.23.5
+pillow==9.5
```

Comment:
We have already downgraded the pillow version from #255.
Please check if there is a conflict.